site/gulpfile.js

```diff
@@ -147,14 +153,21 @@ gulp.task('webserver', () => {
         }));
 });
 
+gulp.task('reload-swag', cb => {
+    delete require.cache[require.resolve('./get-data')];
+    ({ swagImages, swagList } = require('./get-data'));
+    cb();
+});
+
 gulp.task('watch', () => {
-    gulp.watch('src/pug/**/*.pug', gulp.parallel('pug'));
-    gulp.watch('src/styl/**/*.styl', gulp.series('clean', 'styl', 'cachebust'));
-    gulp.watch('src/js/*.js', gulp.series('clean', 'js', 'cachebust'));
+    gulp.watch('../data.json', gulp.series('reload-swag', 'swag-img', 'cachebust', 'pug'));
+    gulp.watch('src/pug/**/*.pug', gulp.series('pug'));
+    gulp.watch('src/styl/**/*.styl', gulp.series('clean:styl', 'styl', 'cachebust'));
+    gulp.watch('src/js/*.js', gulp.series('clean:js', 'js', 'cachebust'));
```

Comment:
We can just run cachebust once at the beginning since the manifest won't change 🤔

Reply:
What do you call the beginning? `swag-img`, `styl` and `js` tasks can change the manifest content, right?

Reply:
The manifest shouldn't change when watching because cachebusting should be disabled in development

Reply:
I dropped calls to cachebust for the watch `js` and `css` and it should do the trick as `cachebust` is ran by the `build` task so it has already ran when `watch` task starts. I keep if for watch `data` as it should react when adding / removing a swag item.